packages/notifications/src/Notification.php

```diff
@@ -88,6 +88,11 @@ public function toArray(): array
     public static function fromArray(array $data): static
     {
         $static = static::make($data['id'] ?? Str::random());
```

Comment:
Shouldn't the `make()` method instantiate a new CustomNotification from the container, instead? I think thats the original idea behind binding it to the container

Reply:
it does get the CustomNotification, this is not wrong, but as it only gets it here, the `CustomNotification::fromArray` is not called at all

Reply:
in `Livewire\Notifications`
```php
// before fix
Notification::fromArray
   > `make()` return `CustomNotification`
   > set properties to `CustomNotification` with Notification's definition
   > return to lw

$this->push into collection
```

```php
// after fix
Notification::fromArray
   > `make()` return `CustomNotification`
   > call `CustomNotification::fromArray`
        > call `parent:fromArray`
        > run `CustomNotification` logic
   > return to lw

$this->push into collection
```

maybe this is clearer?